### PR TITLE
Insert observation attribute

### PR DESCRIPTION
**Pull request by coryverta**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java [hunk 1]
```diff
@@ -866,24 +866,42 @@ public void logObservations(String experimentRunId, List<Observation> observatio
...
           .forEach(
               observation -> {
+                String kvSql =
+                    "INSERT INTO keyvalue"
+                        + " (entity_name, field_type, kv_key, kv_value, value_type, experiment_run_id)"
+                        + " VALUES (:entity_name, :field_type, :kv_key, :kv_value, :value_type, :experiment_run_id)";
+                final long attributeId =
+                    session
+                        .createNativeQuery(kvSql)
+                        .setParameter("entity_name", ExperimentRunEntity.class.getSimpleName())
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java]: missing for observations

File: backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java [hunk 2]
```diff
@@ -866,24 +866,42 @@ public void logObservations(String experimentRunId, List<Observation> observatio
...
+                        .setParameter("field_type", ModelDBConstants.OBSERVATIONS)
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java]: seems to be "attributes" 🤷‍♂️

File: backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java (hunk 1 above)
Comment (coryverta) [reply backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java]: Do you mean that observations do not populate this field?

File: backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java (hunk 2 above)
Comment (coryverta) [reply backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java]: :+1:

File: backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java (hunk 1 above)
Comment (conradoverta) [reply backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java]: correct.

Comment (coryverta) [reply backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java]: :+1:

**Pull request merged**